src/ray/gcs/redis_accessor.h

```diff
@@ -132,6 +132,91 @@ class RedisTaskInfoAccessor : public TaskInfoAccessor {
   TaskSubscriptionExecutor task_sub_executor_;
 };
 
+/// \class RedisNodeInfoAccessor
+/// RedisNodeInfoAccessor is an implementation of `NodeInfoAccessor`
+/// that uses Redis as the backend storage.
+class RedisNodeInfoAccessor : public NodeInfoAccessor {
+ public:
+  explicit RedisNodeInfoAccessor(RedisGcsClient *client_impl);
+
+  virtual ~RedisNodeInfoAccessor() {}
+
+  /// Register local node to GCS synchronously.
+  ///
+  /// \param node_info The information of node to register to GCS.
+  /// \return Status
```

Comment:
Change these method comments to `/// See super class.`, or just remove them.

Reply:
done